src/vs/editor/contrib/stickyScroll/browser/stickyScroll.css

```diff
@@ -32,6 +32,10 @@
 	cursor: pointer;
 }
 
+.monaco-editor .sticky-line-root.focus {
```

Comment:
Suggestion to try the `:focus` pseudo class (https://developer.mozilla.org/en-US/docs/Web/CSS/:focus). That's managed by the browser so you don't need to do much yourself, esp you don't need to add/remove the classname via JS-code